### DISCUSSION TITLE
lotus-miner sector extension failed..

### DISCUSSION THREAD

**Discussion by JshFeel** [filecoin-project/lotus#10598] [General Q&A]
You entered a command to extend the Lotus Miner expiry sector.

- Command entered.

lotus-miner sectors renew --from 1608091 --to 1555200

but the next day
The status value of SubmitWindowedPoSt is shown below.

1. ErrIllegalArgument
2. ErrForbidden

Unable to attempt sector extension because the SubmitWindowedPost failed, resulting in Adjusted Power of 0 TB.

other than the above command
another command is
I can't try because it prints nothing to extend .

What should I do?

**Comment by Derek-zd** [comment]
Have to send renrew message ？
Does the command use "--really-do-it" "--tolerance" ?

**Reply by JshFeel** [reply]
I used --really-do-it for sector extension, but SubmitWindowPost failed the next day.
